app/controllers/CreateUserController.js

```diff
@@ -0,0 +1,71 @@
+var blueprint   = require ('@onehilltech/blueprint')
+    //, database  = require ('@onehilltech/blueprint-mongodb')
+    , util      = require ('util')
+    , request   = require ('superagent')
+    , http = require('http')
+
+    ;
+var apiAddr = 'http://localhost:5000';
+
+function UserCreationController () {
```

Comment:
I would also change the name of the Controller to simply UserController since we will probably need to add methods to it later for retrieving Users, etc.